roles/openshift_sdn_master/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install openshift-sdn-master
+  yum:
+    pkg: openshift-sdn-master
+    state: installed
+
+- name: Configure openshift-sdn-master settings
+  lineinfile:
+    dest: /etc/sysconfig/openshift-sdn-master
+    regexp: '^OPTIONS='
+    line: "OPTIONS=\"-v={{ openshift_sdn_master_debug_level }}\""
+  notify:
+  - restart openshift-sdn-master
+
+- name: Set openshift-sdn-master facts
+  include: "{{ role_path | dirname }}/openshift_common/tasks/set_facts.yml"
+  facts:
+  - { section: sdn-master, option: debug_level, value: "{{ openshift_sdn_master_debug_level }}" }
```

Comment:
@detiber consider staying in yaml instead of going to JSON. I know that this is valid, but I've noticed that ansible error messages are pretty poor when it comes to JSON, whereas with yaml it'll point out the error right away.

I notice it on this diff: https://github.com/twiest/openshift-ansible/commit/de2fa14b28bf34207dcc21b698cea441d6e918bc

In JSON, it was giving me a non-descript error. When I changed the syntax to YAML, it pointed out the exact error. I've decided I'm only going to use YAML now so that we get the best errors possible.

I won't block the PR on this, let me know your thoughts.